include/la/avdecc/controller/internals/avdeccControlledEntityDynamicModel.hpp

```diff
@@ -94,7 +94,7 @@ struct ClockSourceNodeDynamicModel
 struct MemoryObjectNodeDynamicModel
 {
 	entity::model::AvdeccFixedString objectName{};
-	std::uint64_t length{ 0u };
+	entity::model::MemoryObjectLength objectMemoryLength{};
```

Comment:
Currently there is no real advantage - I just wanted it to look like the already implemented getter structs (like StreamInfo and AvbInfo). 

Should I change it back to std::uint64_t?

Reply:
I think we can keep it simple for now, yes.
It was mandatory for the other structs because it's not just one field.
Also, because I'm reworking the enumeration state machine to get information from multiple sources (descriptor/commands) for other fields, I'll probably rework that part as well :)